finagle-partitioning/src/main/scala/com/twitter/finagle/partitioning/KetamaFailureAccrualFactory.scala

```diff
@@ -6,7 +6,8 @@ import com.twitter.finagle.client.Transporter
 import com.twitter.finagle.liveness.{FailureAccrualFactory, FailureAccrualPolicy}
 import com.twitter.finagle.service.{ReqRep, ResponseClassifier}
 import com.twitter.finagle.stats.StatsReceiver
-import com.twitter.finagle._
+import com.twitter.finagle.{param => finagleparam, _}
```

Comment:
nits: I checked a few files, import alias usually being camel cases.

Reply:
I'll get to that tomorrow.

Reply:
Narrator: he didn't.

Reply:
Did now though